appinventor/components/src/com/google/appinventor/components/runtime/ChatBot.java

```diff
@@ -252,12 +262,25 @@ public void Converse(final String question) {
     AsynchUtil.runAsynchronously(new Runnable() {
       @Override
       public void run() {
-        performRequest(uuid, question, null);
+        performRequest(uuid, question, null, false);
       }
-      });
+    });
+  }
+
+  @SimpleFunction(description = "Create an Image. Note: Only Gemini is currently supported. " +
+    "Do not specify a model in order to get the most up-to-date " +
+    "model to use.")
+  public void CreateImage(final String prompt) {
```

Comment:
I think we should rename this parameter to `description` to match `CreateImage` in ImageBot.